packages/processhacker.vm/tools/chocolateyinstall.ps1

```diff
@@ -0,0 +1,21 @@
+$ErrorActionPreference = 'Stop'
+Import-Module vm.common -Force -DisableNameChecking
+
+try {
+  $toolName = 'ProcessHacker'
+  $category = 'Utilities'
+
+  $shortcutDir = Join-Path ${Env:TOOL_LIST_DIR} $category
+  $shortcut = Join-Path $shortcutDir "$toolName.lnk"
+  $executablePath = Join-Path ${Env:ProgramFiles} "\Process Hacker 2\ProcessHacker.exe" -Resolve
+  Install-ChocolateyShortcut -shortcutFilePath $shortcut -targetPath $executablePath -RunAsAdmin
+  VM-Assert-Path $shortcut
+
+  # Delete Desktop shortcut
+  $desktopShortcut = Join-Path ${Env:Public} "Desktop\Process Hacker 2.lnk"
+  if (Test-Path $desktopShortcut) {
```

Comment:
[nitpick] This `if` is not needed as the next command has the `-ea 0`that is the same as `-ErrorAction SilentlyContinue`. We have currently some packages with this if and some without. I would remove it for simplicity.